internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -256,6 +258,61 @@ func convertHTTPHealthCheck(hc *manifest.HealthCheckArgsOrString) template.HTTPH
 	return opts
 }
 
+func (s *LoadBalancedWebService) convertNetworkLoadBalancer() (*template.NetworkLoadBalancer, error) {
```

Comment:
I wonder if we should return a struct here that has all the NLB related functionality to make the `Template` method more readable:
```go
type nlbConfig struct {
   settings *template.NetworkLoadBalancer
   
   // If a domain is associated these values are not empty.
   dnsRole           *string
   dnsName           *string
   certManagerLambda string
}

func (s *LoadBalancedWebService) convertNetworkLoadBalancer() (nlbConfig, error)

```

Reply:
Updated boss!